src/testdir/test_functions.vim

```diff
@@ -4106,7 +4106,8 @@ func Test_isabsolutepath()
     call assert_true(isabsolutepath('A:\Foo'))
     call assert_true(isabsolutepath('A:/Foo'))
     call assert_false(isabsolutepath('A:Foo'))
-    call assert_false(isabsolutepath('\Windows'))
```

Comment:
Hm, wondering if anybody relies on the current behaviour

Reply:
Good question.
+ It turns out `cmd.exe` has always support this (`command.com` didn't) but was seldom used.
  `cmd.exe` use of forward slash `/` as path separator is still limited and discoraged.
  Thus, old vim scripts surely didn't use path with implicit drives.
+ The problem is *powershell*. It supports completely all path notations. And the new *.NET* core
  version is crossplatform (and readily available in all github runners).
  Many people is using linux-like paths for convenience.
  Thus, the new vim scripts will surely rely on this behaviour.

Reply:
any idea to handle it just for powershell? it will be regression for cmd.exe .